src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/UnitTests/StatisticsServiceTests.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using AutoMapper;
+using DevAdventCalendarCompetition.Repository;
+using DevAdventCalendarCompetition.Repository.Context;
+using DevAdventCalendarCompetition.Repository.Interfaces;
+using DevAdventCalendarCompetition.Repository.Models;
+using DevAdventCalendarCompetition.Services;
+using DevAdventCalendarCompetition.Services.Interfaces;
+using DevAdventCalendarCompetition.Services.Models;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace DevAdventCalendarCompetition.Tests.UnitTests
+{
+    public class StatisticsServiceTests
+    {
+        private readonly Mock<IStatisticsRepository> _statisticsRepositoryMock;
+        private readonly StatisticsService _statisticsService;
+
+        private readonly List<StatisticsDto> _results = new List<StatisticsDto>
+        {
+            new StatisticsDto { CorrectAnswerDateTime = DateTime.MinValue.AddDays(1), WrongAnswerCount = 1, TestNumber = 1 },
+            new StatisticsDto { CorrectAnswerDateTime = DateTime.MinValue.AddDays(2), WrongAnswerCount = 2, TestNumber = 2 },
+            new StatisticsDto { CorrectAnswerDateTime = DateTime.MinValue.AddDays(3), WrongAnswerCount = 3, TestNumber = 3 }
+        };
+
+        public StatisticsServiceTests()
+        {
+            this._statisticsRepositoryMock = new Mock<IStatisticsRepository>();
+            this._statisticsService = new StatisticsService(this._statisticsRepositoryMock.Object);
+        }
+
+        [Fact]
+        public void GetStatisticsDtoList()
+        {
+            // ARRANGE
+            string userId = "c611530e-bebd-41a9-ace2-951550edbfa0";
+
+            // I assume it is wrong
```

Comment:
why you are assuming it is wrong?

Reply:
bad wording - I assumed that the way I created those tests isn't right. Looks like there is too many Mocks

Reply:
I don't think so. So maybe just remove the comment and I will close PR

Reply:
@BlackMorzan please resolve conflicts